docs/governance/tags-define.md

```diff
@@ -1,83 +1,110 @@
+You must have a **Fabric administrator** role to create and manage tenant-level tags.
 
-## Prerequisites
+You must have a **Fabric administrator** or **Domain administrator** role to create and manage domain-level tags within your assigned domains.
 
-You must have a Fabric administrator role or higher to be able to create, rename, or delete tags.
+## Create tenant-level tags
 
-## Create a set of tags
+This section shows how Fabric administrators create a set of tenant-level tags. These tags are available across the entire Fabric tenant. 
 
-This section shows how to create a set of one or more tags.
+1.  Open the [admin portal](../admin/admin-center.md%23how-to-get-to-the-admin-portal) and select **Tags**.
+    The **Tags** tab opens. All currently defined tenant-level tags are listed on the tab.
+    
+1.  Select **+ New tag**. If no tags have been defined yet, select **Add your first tags**.
+   
+1.  In the dialog that appears, provide names for your new tags. Use comma-separated names to create more than one tag at a time. Tag names can contain letters, numbers, spaces (not at the beginning), and special characters (must also contain letters and/or numbers).
 
-1. Open the [admin portal](../admin/admin-center.md#how-to-get-to-the-admin-portal) and select **Tags**.
+    :::image type="content" source="./media/tags-define/create-tags.png" alt-text="Screenshot showing how to create two new tags.":::
 
-    The **Tags** tab opens. All currently defined tags are listed on the tab.
+1.  When done, select **Create**. The new tags are added to the set of tags.
 
-1. Select **+ New tag**. If no tags have been defined yet, select **Add your first tags**.
 
-1. In the dialog that appears, provide names for your new tags. Use comma-separated names to create more than one tag at a time. Up to 10,000 tags can be created in a tenant.
+## Create domain-level tags
 
-    :::image type="content" source="./media/tags-define/create-tags.png" alt-text="Screenshot showing how to create two new tags.":::
+This section shows how [domain](./domains.md) administrators create and manage tags specific to their assigned domains. These tags are only available for items within workspaces assigned to that domain and sub-domains under it.
+
+### Tag Uniqueness Rules
+
+When creating domain-level tags, consider the following uniqueness rules:
 
-    Tag names can contain:
-    * Letters
-    * Numbers
-    * Spaces (not at the beginning of the tag)
-    * Special characters (the name can't consist solely of special characters; it must also contain letters and/or numbers)
+  1. A tag created at the tenant level cannot be duplicated at the domain level and vice-versa. If you attempt to create a tag that duplicates an existing one at a different scope (tenant or domain), an error message will indicates the existing tag and its scope.
```

Comment:
Line 57 suggestion:

1. A tag created at the tenant level cannot be duplicated at the domain level and vice-versa. If you attempt to create a tag that duplicates an existing one at a different scope (tenant or domain), an error message indicates the existing tag and its scope.